utils/migration_tools/model_registry_to_runs/README.md

```diff
@@ -0,0 +1,76 @@
+# Migrating from the Model Registry to Experiments
+
+This script allows you to copy models and model versions from the Model Registry to Experiments, within the same project.
+
+## Why Migrate?
+Our revamped Web App adds a host of new features to Experiments, including better grouping, and enhanced reporting capabilities. By migrating your models and model versions to Experiments, you can take advantage of these features to better organize and compare your models.
+
+As a part of our focus on performance this year, the `Run` API is also be highly optimized for speed. Using the `Run` object instead of `Model` or `ModelVersion` objects will allow you to take advantage of these optimizations.
+
+This script leverages the new *Group Tags* to organise your models and model versions in Experiments, and *tags* to create saved views separating model metadata runs from training runs. This lets you view all Models and Model Versions in the same table as shown below:
+
+![Organise Models in Experiments](https://neptune.ai/wp-content/uploads/2024/07/MRtoRun.png)
+
+In this screenshot, _Group Tag_, _Custom Run ID_, and _ID_ correspond to the original Model ID, Model Version ID and the new Run ID respectively.
+
+Having all model metadata in Experiments also lets you use Experiment's native [comparisons](https://docs.neptune.ai/usage/tutorial/#compare-the-runs) and [Reports](https://docs.neptune.ai/app/reports/) to compare models and model versions.
+
+
+## Prerequisites
+
+Before using this script, make sure you have the Neptune environment variables set up. For instructions, see the [documentation](https://docs.neptune.ai/setup/setting_credentials/).
+
+Additionally, ensure that the project you want to copy metadata to has already been created.
```

Comment:
```suggestion
Additionally, ensure that you want to copy metadata to an already existing project.
```

Reply:
This prerequisite is not valid anymore. Will remove.